tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleTooling.java

```diff
@@ -99,17 +61,24 @@ else if( !bundleLocationString.contains( installPath ) )
 
             final ModelBuilder<T> modelBuilder = (ModelBuilder<T>) connection.model( modelClass );
 
+            final File depsDir = new File(cacheDir, "deps");
```

Comment:
Hey @AndyWu2015  can we cache this extraction some how?  I don't want to extract these each time.  That will be too slow.